test/applications_integration_test.go

```diff
@@ -111,7 +111,7 @@ func waitForTransaction(transactionId string) error {
 func iBuildAnApplicationTransaction(
 	operation, approvalProgram, clearProgram string,
 	globalBytes, globalInts, localBytes, localInts int,
-	appArgs, foreignApps, appAccounts string) error {
+	appArgs, foreignApps, foreignAssets, appAccounts string) error {
```

Comment:
This is only added to the unit tests at the moment, so it shouldn't be added here.